test/oli_web/controllers/api/payment_controller_test.exs

```diff
@@ -148,6 +149,26 @@ defmodule OliWeb.PaymentControllerTest do
 
       reset_test_payment_config()
     end
+
+    test "download .txt file with the last payment codes created", %{conn: conn} do
```

Comment:
What if we don't create any codes? Would be great to test that case, along with the case where we don't send a `count` query param, wdyt?